package.json

```diff
@@ -19,7 +19,7 @@
   },
   "peerDependencies": {
     "expo-linking": "*",
-    "react": "^18.0.0",
```

Comment:
I didn't know you could specify versions like this, but according to https://semver.npmjs.com/ you can 🤷‍♂️

Reply:
@janmeier Yeah I thought one had to do ^18 | ^19?

Reply:
Yep, that works as well